### DISCUSSION TITLE
Collections - Hide items from Library view

### DISCUSSION THREAD

**Discussion by frasderp** [gotson/komga#1561] [Ideas]
### Describe your suggested feature

Similar to Plex, I would like the option to hide items in a Collection from the Library view. I have a number of related comic runs that clutter the library, and would prefer to group them as a result (spin-offs etc).

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Dupe of #230 #706

**Reply by flamesky80** [reply]
This feature is very useful for me too, if you could add it, we would really appreciate it.

I'm not a programmer, and here's just a thought. If it's wrong, please ignore it.

First, Komga definitely knows how many collections it has and what their names are. Otherwise, it couldn't show the information in the 'Collections' tab.
Second, every comic definitely knows whether it belongs to a collection, so it can be shown in the collection page. Alternatively, Komga can query the database and add a tag to those comics.

Now, when a user clicks the Library tab:
1. Query the database and get a list of all the collections, such as [C1, C2, C3, ...].
2. Query the database and get a list of all the comic books and series, such as [a, b, c, d, e, f, ...].
3. Find out the list of comics that have a collection tag, such as [a, d, f, ...].
4. Subtract the list of comics with a collection tag from the total list: [a, b, c, d, e, f, ...] - [a, d, f, ...] = [b, c, e, ...].
5. Finally, the Library tab should show all the collections and the remaining comics, such as [C1, C2, C3, ..., b, c, e, ...].

@gotson

**Reply by gotson** [reply]
@flamesky80 stop tagging me directly

**Reply by flamesky80** [reply]
> @flamesky80 stop tagging me directly

Sorry for bothering you. As a big fan of Komga, I just want to share some of my personal experiences, and I do hope it becomes more popular with a better user experience.

**Comment by RickSWilliams** [comment]
I’d just like to add I would love this feature. I’ve got a couple series I follow that have an absurd number of one shots and minis. It ballon’s my library view by almost 75% and makes it very difficult to navigate. This would fix that issue perfectly.

**Comment by a-elkaim** [comment]
Same, I would love for this feature to be added !

**Comment by JamesDAdams** [comment]
up

**Comment by yamanobe96** [comment]
I'm also asking for this feature to be added

I'm also asking for this feature to beFor example, there are three manga in the series Makibao. There are two types of manga called Rurouni Kenshin. I would like to display these titles together in a library.
The ideal form would look something like this: I would like a browser function that allows me to organize the items included in a collection as a collection. I'm looking for such a feature added
![Komga - Google Chrome 2024_02_04 17_40_18 - コピー](https://github.com/gotson/komga/assets/46581513/11ec9326-23fb-48f3-81ca-de0c216aa642)
![Komga - Google Chrome 2024_02_04 17_40_18 - コピー (2)](https://github.com/gotson/komga/assets/46581513/53dde071-4422-4d2b-8f1a-842f27b73f15)

**Comment by balthisar** [comment]
Although this is marked as a dupe, this is, in fact, the only extant issue that's open and not closed.

**Comment by yamanobe96** [comment]
please
Please help us resolve this issue
Authors such as CLAMP, Fujiko F, Fujio, and Osamu Tezuka who have released multiple series of two books in one book will have their libraries crowded unless they are compiled.
Bobobobobobobobo and Makibao are placed in separate locations in the library because the initials of the titles change in the sequels.

**Comment by theradiostar** [comment]
Would also appreciate the addition of this feature.

**Comment by MuffinOfPotato** [comment]
Here is a relevant mockup I made (before & after) in support of this feature before I was informed my idea was a dupe. Would love to see something like this become an addition to NextUI.

<img width="1663" height="933" alt="image" src="https://github.com/user-attachments/assets/446f584a-eb2b-49b0-946a-90cecb961d94" />
<img width="1664" height="898" alt="image" src="https://github.com/user-attachments/assets/fac4b714-a77a-4db7-b537-2a0b9a8a1cff" />

The idea here is a basic toggle (depicted as a folder icon here) that would enable a view that removes any series that Komga identifies as belonging to a collection, which would then allow series and collections to be ordered with each other.
